src/lib/components/FormModel/children/Form/UI/DateTimePicker/date-time-picker.component.js

```diff
@@ -66,9 +66,10 @@ export const DateTimePicker = props => {
     let maxHours;
     let maxMinutes;
 
-    const timeFormat = /ˆ\d{1,2}:\d{2}$/;
+    // Regex to match the format "12:##:##"
+    const timeFormat = /\d{1,2}:\d{2}:\d{2}$/;
```

Comment:
This comment doesn't seem right? It looks more like:

```suggestion
    // Regex to match the format "##:##:##" or "#:##:##"
    const timeFormat = /\d{1,2}:\d{2}:\d{2}$/;
```

(i.e. one or two digits, a colon, two digits, a colon, two digits)

Is that the intention?

Also, why was the `^` at the start removed, but the `$` at the end preserved? I'd assume you'd either want both or neither? (And my guess would be both?)

Reply:
You're correct about the comment (I didn't write the original, just added the seconds piece). I can update it.

The reason I removed the ^ is, frankly, it didn't match with it and I am not a regex wizard

Reply:
`^` says "the given string should start with the following characters", and `$` says "the given string should end with the preceding characters".

So it does match it now?